lib/commit_after_review.js

```diff
@@ -0,0 +1,73 @@
+function commitAfterReview(commit, review, logger) {
+  let commitIndex = commit.length === 1
+    ? 0 : commit.length - 1;
+  let reviewIndex = review.length === 1
+    ? 0 : review.length - 1;
+
+  let lastCommit = commit[commitIndex].commit;
+  let lastReview = review[reviewIndex];
+
+  let commitInfo = getDate(lastCommit.committedDate);
```

Comment:
I don't think we need a date object, or get the years/minutes out. ISO date strings are in lexical orders so a simple `lastReview .publishedAt < lastCommit.committedDate` is all we need.

Reply:
I did not know that!